share/models/core.py

```diff
@@ -183,9 +186,9 @@ def store_data(self, doc_id, data, source, app_label):
         )
 
         if created:
-            logger.debug('Newly created RawData for document {} from {}'.format(doc_id, source))
+            logger.debug('Newly created RawData for document %s from %s', doc_id, source)
```

Comment:
Ideally yes? It saves a touch of time as `.format` is always run. Using the `%s` will only be formatted if the log level matches.
The most important place to use this is when logging query sets and the like